<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="toolTipDuration">
   <number>-19</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GUI</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="hotkeyCheckbox">
        <property name="toolTip">
         <string>Global hotkey to minimize to and restore from system tray</string>
        </property>
        <property name="text">
         <string>Global &amp;hotkey:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="clearHotkeyButton">
        <property name="text">
         <string>&amp;Clear</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KeySequenceEdit" name="hotkeyInput"/>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="systrayCheckbox">
        <property name="toolTip">
         <string>Minimize to system tray when closed, instead of exiting the application</string>
        </property>
        <property name="text">
         <string>Close to &amp;system tray</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>External &amp;editor:</string>
        </property>
        <property name="buddy">
         <cstring>externalEditorInput</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="externalEditorInput">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If the command isn't an absolute path, the PATH environment variable will be used.&lt;/p&gt;&lt;p&gt;Optionally, you can use {file} and/or {search_term}. If you don't use {file}, the file name will be appended.&lt;/p&gt;&lt;p&gt;Examples:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;notepad&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;notepad.exe&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;c:\windows\system32\notepad.exe&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;notepad {file}&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;gvim +/{search_term}&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>hotkeyInput</zorder>
     <zorder>hotkeyCheckbox</zorder>
     <zorder>systrayCheckbox</zorder>
     <zorder>clearHotkeyButton</zorder>
     <zorder>label</zorder>
     <zorder>externalEditorInput</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>&amp;Repositories</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="ReposTreeWidget" name="reposTree">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Color</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Path</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReposTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>noteeds.gui.settings</header>
  </customwidget>
  <customwidget>
   <class>KeySequenceEdit</class>
   <extends>QKeySequenceEdit</extends>
   <header>noteeds.gui.settings</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>systrayCheckbox</tabstop>
  <tabstop>hotkeyCheckbox</tabstop>
  <tabstop>hotkeyInput</tabstop>
  <tabstop>clearHotkeyButton</tabstop>
  <tabstop>externalEditorInput</tabstop>
  <tabstop>reposTree</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
